### PR TITLE
Make tests PEP8 compliant

### PR DESCRIPTION
**Pull request by gui-miotto**

### CODE REVIEW

File: test/test_pipeline/implementations/test_util.py [hunk 1]
```diff
@@ -5,11 +5,12 @@
 
 from autosklearn.pipeline.implementations.util import softmax
 
+
 class UtilTest(unittest.TestCase):
     def test_softmax_binary(self):
-        df = np.array([-40.00643897, 34.69754581, 23.71181359 -29.89724287,
+        df = np.array([-40.00643897, 34.69754581, 23.71181359 - 29.89724287,
```
Comment (gui-miotto) [inline test/test_pipeline/implementations/test_util.py:11]: Is this supposed to really be a subtraction or there is a comma missing here?
(Same question for line 13)

**Comment by gui-miotto** [comment]
`test_metalearning.py`  uses two functions (`calc_meta_features` and `calc_meta_features_encoded` ) whose definitions I couldn't find. I think this didn't cause errors before due to star imports.
What should we do here?

**Comment by gui-miotto** [comment]
`test/test_pipeline/components/dummy_components/dummy_component_import.py` is 100% PEP8 deviating. :-D
Is this file still necessary?

File: test/test_pipeline/implementations/test_util.py (hunk 1 above)
Comment (mfeurer) [reply test/test_pipeline/implementations/test_util.py:11]: There is a comma missing

**Comment by mfeurer** [comment]
> test_metalearning.py uses two functions (calc_meta_features and calc_meta_features_encoded ) whose definitions I couldn't find. I think this didn't cause errors before due to star imports.
What should we do here?

Apparently, that unit test was skipped. You can find those functions in `autosklearn.smbo`

**Comment by mfeurer** [comment]
> test/test_pipeline/components/dummy_components/dummy_component_import.py is 100% PEP8 deviating. :-D
> Is this file still necessary?

Yes. Please put a `# noqa` there to ignore the PEP8 error(s).

File: test/test_pipeline/implementations/test_util.py (hunk 1 above)
Comment (gui-miotto) [reply test/test_pipeline/implementations/test_util.py:11]: OK

**Comment by gui-miotto** [comment]
> > test_metalearning.py uses two functions (calc_meta_features and calc_meta_features_encoded ) whose definitions I couldn't find. I think this didn't cause errors before due to star imports.
> > What should we do here?
> 
> Apparently, that unit test was skipped. You can find those functions in `autosklearn.smbo`

Ok. They have slightly different names over there. That's why I couldn't find them. For instance `_calculate_metafeatures` instead of `calc_meta_features`.

**Comment by gui-miotto** [comment]
> > test/test_pipeline/components/dummy_components/dummy_component_import.py is 100% PEP8 deviating. :-D
> > Is this file still necessary?
> 
> Yes. Please put a `# noqa` there to ignore the PEP8 error(s).

ok

**Comment by gui-miotto** [comment]
Last one. `test/test_metalearning/test_metalearning.py` uses some constants like `ACC_METRIC` and `AUC_METRIC` (line 36 onwards). I couldn't find the definition of those as well.

**Comment by mfeurer** [comment]
> Last one. test/test_metalearning/test_metalearning.py uses some constants like ACC_METRIC and AUC_METRIC (line 36 onwards). I couldn't find the definition of those as well.

Whoa, that's super-old code. You can make the constants strings for now to fix PEP8.

**Comment by gui-miotto** [comment]
@mfeurer , this is ready for review

**Pull request merged**